corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -128,9 +128,17 @@ <h1 class="page-title"><%- title %></h1>
 </script>
 
 <script type="text/template" id="form-menu-template">
-  <div tabindex="0" id="form-menu-dropdown">
-     <i class="fa fa-bars dropdown-toggle" data-toggle="dropdown"></i>
-     <ul class="dropdown-menu dropdown-menu-right noprint-sub-container">
+  <div id="form-menu-dropdown">
+    <a class="dropdown-toggle"
+       title="{% trans 'Dropdown toggle' %}"
+       role="button"
+       href="javascript:void(0);"
+       data-toggle="dropdown"
+       aria-haspopup="true"
+       aria-expanded="false">
+      <i class="fa fa-bars" style="color: #fff"></i>
```

Comment:
Can the styling be in stylesheet?

Reply:
I don't think this markup is right. The font color styling caught my eye, it's rare to need to set font color in our codebase. Looking at the [dropdown docs](https://getbootstrap.com/docs/3.3/components/#btn-dropdowns), my guess is that the `a` should be a button and the `form-menu-dropdown` div may need to be a `btn-group`.

Reply:
Fixed. [754b612](https://github.com/dimagi/commcare-hq/pull/32995/commits/754b61277d0663d7ffcc354990922269931cd251)